cves/kernel/CVE-2016-2184.yml

```diff
@@ -151,11 +162,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
-autodiscoverable:
+  answer: 
+    The bug was found using the USB-fuzzing frramework from SergeJ Shumilo. This
+    found out that the crash was caused by a zero value for bNumEndpoints.
```

Comment:
Typo in "frramework". Also I would suggest replacing the variable name with a high-level description of what that variable represents.